requirements/base.txt

```diff
@@ -41,7 +41,6 @@ suds==0.4
 tinys3==0.1.11
 unipath>=1.1,<=2.0
 vobject==0.9.1
-wagtail==1.8.1
```

Comment:
@chosak that was actually my bad -- it was left over from a rebase conflict. and yes, i can do that!